GenXIntrinsics/lib/GenXIntrinsics/GenXSPIRVReaderAdaptor.cpp

```diff
@@ -249,7 +249,6 @@ static ArgKind mapSPIRVTypeToArgKind(SPIRVType Ty) {
   case SPIRVType::Other:
     return ArgKind::General;
   case SPIRVType::None:
-  default:
```

Comment:
Removal of this line will trigger warning if code is compiled with MSVC: https://godbolt.org/z/PGfPTP. To avoid this I propose to either remove `None` case instead of `default` or to add another `llvm_unreachable` after whole switch. Second option will still allow to compiler to emit warnings if enum is changed so personally I would prefer it.

Reply:
@aus-intel  Thanks for the review.
So this is what worked with clang. 
Removing the None case didn't work. Clang complained about None missing case.
Not putting the break after the None case, clang complains about "enumeration case not handled".
Is this acceptable to you?

Reply:
I think proposed way is fine. Please squash commits into one and we will merge this PR.

Reply:
Done. Thanks.